elytron/EAP7-1856-Bearer-Token-Authorization-Timeout.adoc

```diff
@@ -0,0 +1,120 @@
+= [Preview] Bearer token timeout configurability will be added to WildFly's Elytron subsystem.
```

Comment:
This needs to category front matter -- see the first seven lines of https://raw.githubusercontent.com/wildfly/wildfly-proposals/main/design-doc-template.adoc.

Reply:
done